internal/datanode/data_sync_service.go

```diff
@@ -372,3 +390,56 @@ func (dsService *dataSyncService) getSegmentInfos(segmentIDs []int64) ([]*datapb
 	}
 	return infoResp.Infos, nil
 }
+
+func (dsService *dataSyncService) getDmlChannelPositionByBroadcast(ctx context.Context, channelName string, ts uint64) ([]byte, error) {
```

Comment:
why we get position by broadcase?
msgstream already support GetLatestMsgID?

Reply:
Interesting. @wayblink I left a TODO here.